lookup-table.yaml

```diff
@@ -177,6 +190,17 @@
     - packageId: EmilyPloszaj.chime
       loaders:
         - fabric
+- id: quilt-loading-screen
+  alternativeNames: ~
+  tags:
+    - cosmetic
+    - decoration
+    - miscellaneous
+    - utility
+  packages:
+    - packageId: Emmaffle.quilt-loading-screen
```

Comment:
Either this packageId is wrong, or the folder `/manifests/E/emaffle` you made is wrong

Reply:
The folder still seems to be `emaffle`, and this is still `Emaffle`. Which one is it?

Reply:
The folder is `Emmafle`(one `f`)  while this is `Emmaffle` (double `f`). Which one is it?